### ISSUE TITLE
A random thought... or not?

### ISSUE THREAD

**Issue by roelvan** [feathersjs/feathers#311]
Hi guys,

Not sure where to post this…

This idea suddenly popped into my head:

How about we make a cool demo app where people new to the framework can learn from and can see its cool abilities. Something different than the usual todo, chat,… kind of thing.   
**What I had in mind:**
A while ago I discovered [svgporn.com](http://svgporn.com/), basically a website full of high quality SVG logo’s for developers (bummer, no real porn…).
We could use those to make some kind of mini-game: _How good of a developer are you?_  
**How it could work:**
A user clicks: “start the test”. After a short countdown the first random logo appears and another clock starts to count down from eg. 60 seconds. There is one input field where the user has to correctly guess the framework or dev tool. He can either be correct, wrong or he can pass the logo. Correct = extra 5 seconds, every wrong guess = minus 3 seconds, pass = minus 5 seconds.
The goal of the game is to survive as long as possible. In the beginning we show easy ones (github, html5,…) but the more the user progresses the harder it gets. (infer, Rollup.js,…).  
**Why this?**
- it would be a great learning experience: a combo of realtime, rest, frontend, social authentication… resulting into a simple but entertaining product. We could make some video tutorials or blog posts about the process.
- It could give feathers.js some exposure, we brand the game like this: “Built with Feathers.js, learn how.”.
- Every developer would like to know how many dev tools he could guess and compare it to his peers. There could be tweets like: “I just survived 1min15s on How good of a developer are you. Try to beat me here www.…”
- Could be shared on product hunt. Not the end goal though..

To summarize, Feathers gets a new demo app, might have some online exposure, new tutorials or blog posts, a fun game to play, … 

**To wrap up:**
I could help with:
- Design, css, concept.
- Video tutorials: I have a photography, video, animations,… background.

As I am new to Feathers (I come from Meteor) I would need help with:
- backend in feathers.js

Please let me know what you think and if you would be willing to help. Let’s make this a cool Feathers community effort.

Any ideas/opinions are welcome…

**Comment by ekryski** [comment]
@roelvan we would be more than happy to link this in the docs as an example if some people decide to build it but the core team is focused on the [chat example](https://github.com/feathersjs/feathers-chat). We put a lot of thought into what example we should use as the "official" one and a chat app is the most natural fit because it actually makes sense for it to be real-time but it can also be expanded for other things (which is currently happening).

You can read more [on the original issue](https://github.com/feathersjs/feathers/issues/163).

Honestly, I'm not sure where the best place is to post this as well. We only want to keep issues open if the core team plans on addressing them. Maybe we can leave this open for a bit to see if anyone else is interested.

@roelvan another solution might be to just try and grab a few buddies and hack something out. We are more than happy to have submissions/examples to put in [the showcase](http://docs.feathersjs.com/why/showcase.html).

**Comment by roelvan** [comment]
Thanks for the reply. I guess it is better to close the issue (as it is not really one) and look for a better channel to brainstorm about this. Don't want to pollute the issues too much.
Thanks anyway!
